README.md

```diff
@@ -0,0 +1,30 @@
+# Honeycomb Tracing for Faraday
+
+[![Build Status](https://travis-ci.org/honeycombio/faraday-honeycomb.svg?branch=master)](https://travis-ci.org/honeycombio/faraday-honeycomb)
+[![Gem Version](https://badge.fury.io/rb/faraday-honeycomb.svg)](https://badge.fury.io/rb/faraday-honeycomb)
+
+This package makes it easy to instrument your Faraday requests in your ruby application to send useful events to [Honeycomb](https://www.honeycomb.io), a service for debugging your software in production.
+- [Usage and Examples](https://docs.honeycomb.io/getting-data-in/beelines/ruby-beeline/)
```

Comment:
This is the only bit that I'm unsure on. It's not specific to this gem but might still be useful for people. We don't really have any other place to send them though

Reply:
Yeah, seems good for now though. As far as I know, the idea of people using these instrumentations without the beeline is theoretical for now — if it becomes popular or people get confused, maybe we can break out separate docs for the specific instrumentations.